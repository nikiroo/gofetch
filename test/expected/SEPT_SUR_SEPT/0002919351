   QUATRE COMMUNES À FACILITÉS ONT ENVOYÉ LEURS CONVOCATIONS APRÈS 
                         CELLE DU GOUVERNEUR                       

  [Belgique] 26-09-2018 19:07

  o Reference: 0002919351
  o News link: http://s.7s7.be/3480023
  o Source link: 


    Par: rédaction
    
    26/09/18 - 19h07 Source: Belga    © belga.
    
    Les  communes à facilités de Linkebeek, Craainhem, Drogenbos et
    Wezembeek-Oppem   ont   envoyé   leurs   propres   convocations
    électorales  en  vue  du  scrutin  du 14 octobre, deux semaines
    après   celles   qui   avaient   été  envoyées,  uniquement  en
    néerlandais,  par le gouverneur du Brabant flamand, Lodewijk De
    Witte,  à  la  demande  de  la  ministre  flamande des Affaires
    intérieures, Liesbeth Homans.
    
    A  Rhode-Saint-Genèse,  le  collège  a décidé de faire imprimer
    ces  documents  en  français  pour  les  francophones  qui  ont
    demandé  leurs  documents  administratifs  dans  la  langue  de
    Voltaire  pour  quatre  ans, et en néerlandais pour les autres.
    Il  n'a pas encore décidé de leur envoi. La commune de Wemmel a
    décidé  de  ne pas envoyer ses propres convocations disponibles
    au guichet de la commune.
    
    L'ensemble  des  bourgmestres  des  communes  à facilités de la
    périphérie  ont exprimé leur désapprobation au sujet de l'envoi
    de convocations électorales en néerlandais par le gouverneur.
    
    Mme  Homans  (N-VA)  s'en  tient  à la règle décrétale flamande
    selon  laquelle  les  convocations doivent être envoyées à tous
    les  électeurs en néerlandais. A charge des francophones qui le
    souhaitent  de  demander  ensuite  l'envoi d'une convocation en
    français.
    
    Les  bourgmestres  des communes à facilités s'en tiennent quant
    à  eux  à  un  arrêt  du  conseil  d'Etat  pris en 2014 dans le
    dossier  es  bourgmestres  non nommés et à un autre plus récent
    qui  précisent  que  les  habitants francophones des communes à
    facilités  peuvent  demander  la  traduction  systématique  des
    documents  administratifs,  une  seule fois pour les quatre ans
    qui suivent.
    
    Une   grande   majorité  de  francophones  l'ont  fait  jusqu'à
    présent. A Crainhem, ils sont ainsi près de 8.000 dans ce cas.
    
    A   Wemmel,   l'administration  communale  s'est  contentée  de
    l'envoi  d'une  lettre aux francophones, leur précisant qu'elle
    n'est  pas  d'accord  avec  l'envoi  de  la  convocation par le
    gouverneur,  et  que s'ils le souhaitent, ils peuvent se rendre
    à  la  commune pour solliciter une traduction de la convocation
    envoyée par Lode De Witte.



