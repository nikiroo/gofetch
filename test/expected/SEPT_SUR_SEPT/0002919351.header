0Quatre communes à facilités ont envoyé leurs convocations après celle du gouverneur	null/SEPT_SUR_SEPT/0002919351		70
i  [Belgique] 26-09-2018 19:07
i
i    Les  communes à facilités de Linkebeek, Craainhem, Drogenbos et
i    Wezembeek-Oppem   ont   envoyé   leurs   propres   convocations
i    électorales en vue du ...
i
