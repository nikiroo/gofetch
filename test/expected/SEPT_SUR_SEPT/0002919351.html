<!DOCTYPE html>
<html>
<head>
  <meta http-equiv='content-type' content='text/html; charset=utf-8'>
  <meta name='viewport' content='width=device-width, initial-scale=1.0'>
  <style type='text/css'>
    body { margin: 1em 15%; }
  </style>
</head>
<body>
<div class='story'>
	<h1>Quatre communes à facilités ont envoyé leurs convocations après celle du gouverneur</h1>
	<div class='details'>([Belgique] 26-09-2018 19:07)</div>
	<br/>
    <ul>
        <li>News link: <a href='http://s.7s7.be/3480023'>http://s.7s7.be/3480023</a></li>
        <li>Source link: <a href=''></a></li>
    </ul>
	<br/>
	<div class='content' style='text-align: justify'>
		Par: rédaction<br/><br/>26/09/18 - 19h07 Source: Belga    © belga.<br/><br/>Les communes à facilités de Linkebeek, Craainhem, Drogenbos et Wezembeek-Oppem ont envoyé leurs propres convocations électorales en vue du scrutin du 14 octobre, deux semaines après celles qui avaient été envoyées, uniquement en néerlandais, par le gouverneur du Brabant flamand, Lodewijk De Witte, à la demande de la ministre flamande des Affaires intérieures, Liesbeth Homans.<br/><br/>A Rhode-Saint-Genèse, le collège a décidé de faire imprimer ces documents en français pour les francophones qui ont demandé leurs documents administratifs dans la langue de Voltaire pour quatre ans, et en néerlandais pour les autres. Il n&#x27;a pas encore décidé de leur envoi. La commune de Wemmel a décidé de ne pas envoyer ses propres convocations disponibles au guichet de la commune.<br/><br/>L&#x27;ensemble des bourgmestres des communes à facilités de la périphérie ont exprimé leur désapprobation au sujet de l&#x27;envoi de convocations électorales en néerlandais par le gouverneur.<br/><br/>Mme Homans (N-VA) s&#x27;en tient à la règle décrétale flamande selon laquelle les convocations doivent être envoyées à tous les électeurs en néerlandais. A charge des francophones qui le souhaitent de demander ensuite l&#x27;envoi d&#x27;une convocation en français.<br/><br/>Les bourgmestres des communes à facilités s&#x27;en tiennent quant à eux à un arrêt du conseil d&#x27;Etat pris en 2014 dans le dossier es bourgmestres non nommés et à un autre plus récent qui précisent que les habitants francophones des communes à facilités peuvent demander la traduction systématique des documents administratifs, une seule fois pour les quatre ans qui suivent.<br/><br/>Une grande majorité de francophones l&#x27;ont fait jusqu&#x27;à présent. A Crainhem, ils sont ainsi près de 8.000 dans ce cas.<br/><br/>A Wemmel, l&#x27;administration communale s&#x27;est contentée de l&#x27;envoi d&#x27;une lettre aux francophones, leur précisant qu&#x27;elle n&#x27;est pas d&#x27;accord avec l&#x27;envoi de la convocation par le gouverneur, et que s&#x27;ils le souhaitent, ils peuvent se rendre à la commune pour solliciter une traduction de la convocation envoyée par Lode De Witte.
	</div>
<hr/>
</div>
</body>
